test/expected/paths.out

```diff
@@ -43,21 +43,21 @@ select jsonb_pretty(postgrest_openapi_spec('{test}')->'paths'->'/products'->'get
   
 -- uses a reference for the 200 HTTP code response
 select jsonb_pretty(postgrest_openapi_spec('{test}')->'paths'->'/products'->'get'->'responses'->'200');
-                    jsonb_pretty                    
-----------------------------------------------------
- {                                                 +
-     "$ref": "#/components/responses/get.products",+
-     "description": "OK"                           +
+                      jsonb_pretty                       
+---------------------------------------------------------
+ {                                                      +
+     "$ref": "#/components/responses/notEmpty.products",+
```

Comment:
Q: Do we have this notEmpty on postgrest Haskell OpenAPI?

Reply:
Also curious, what's the use of empty and maybeEmpty?

Reply:
> Q: Do we have this notEmpty on postgrest Haskell OpenAPI?

No we don't. The response was specified directly instead of using a reference to that response. E.g:

Haskell OpenAPI
```json
"responses": {
  "200": {
    "description": "OK",
    "schema": {
      "items": {
        "$ref": "#/definitions/orders"
      },
      "type": "array"
    }
  }
```

SQL OpenAPI
```json
"responses": {
  "200": {
    "description": "OK",
    "$ref": "#/components/responses/notEmpty.servers"
  },
```

The latter is better to make it reusable. The non-empty response body can be used in `200` responses for `get` and `patch`, for instance.

> Also curious, what's the use of empty and maybeEmpty?

I changed it to those names since they were more accurate for the response body they are returning.

- `nonEmpty`: When a response body exists, e.g. when `200` OK is returned (GET and PATCH could return this)
- `empty`: When there is no response body, e.g. when `204` is returned. Since this doesn't have a response body it can be reused for any endpoint that allows PATCH.
- `mayBeEmpty`: When the response body could be empty or it could exist, e.g. when returning a `201` after inserting a value. If `return=representation` is not specified then it's empty, else it has a body.

---

On a side note the `mayBeEmpty` has a bit of a problem since it's not exactly possible to return, for example, an `application/json` empty response for `201 Created` (it always returns `[]`). But since the key `201` cannot be repeated, then this was my only workaround.